openscap_report/html_report/templates/template_report.html

```diff
@@ -16,6 +16,7 @@
 <style>
     {%- include './css/patternfly.css' -%}
 </style>
+<!-- <link rel="stylesheet" href="https://unpkg.com/@patternfly/patternfly/patternfly.css" crossorigin="anonymous"> -->
```

Comment:
@Honny1 Have you watched the YouTube videos carefully? Code commented out isn't OK, you probably want to remove it.

Reply:
I missed this comment.